src/theme/index.js

```diff
@@ -0,0 +1,32 @@
+import { initializeStyles } from '#rsu/styles';
+
+import purpleAndPinkColors from './purple-and-pink/colors';
+import purpleAndPinkDimens from './purple-and-pink/dimens';
+
+import defaultColors from './default/colors';
+import defaultDimens from './default/dimens';
+
+export const themes = {
+    default: {
+        title: 'Default',
+        colors: defaultColors,
+        dimens: defaultDimens,
+    },
+
+    purpleAndPink: {
+        title: 'Purple & Pink',
+        colors: purpleAndPinkColors,
+        dimens: purpleAndPinkDimens,
+    },
+
+};
+
+export const setTheme = (themeId) => {
+    const theme = themes[themeId];
```

Comment:
Can we get deafult theme if there is no theme from specified themeId?

Reply:
Maybe